src/pages/detail.html

```diff
@@ -0,0 +1,551 @@
+                <option value="11">11</option>
+                <option value="12">12</option>
+                <option value="13">13</option>
+                <option value="14">14</option>
+                <option value="15">15</option>
+              </select>
+            </div>
+            <div class="w-3/4 mb-3">
+              <label for="number" class="block mb-4 text-base uppercase font-medium text-gray-900"
+                data-i18n="detail.number"></label>
+              <div class="flex">
+                <a href=""
+                  class="uppercase font-bold border border-gray-600 py-2 px-3.5 rounded-xl transition-all hover:bg-black hover:text-white mr-3"
+                  data-i18n="detail.buy"></a>
+                <button class="bg-green-500 py-2 px-3.5 rounded-xl mr-3 text-white hover:text-red-300 transition-all">
+                  <i class="fa-solid fa-heart "></i>
+                </button>
+                <button
+                  class="py-2 px-3.5 rounded-xl border border-gray-600  hover:bg-black hover:text-white transition-all">
+                  <i class="fa-solid fa-rotate-right"></i>
+                </button>
+              </div>
+            </div>
+            <div class="flex mb-3 ">
+              <span class="text-[10px] self-end pb-1.5">Shipping & <span class="text-red-600">Returns</span></span>
+              <img src="../../assets/images/detail_media.png" alt="">
+            </div>
+          </div>
+        </div>
+
+        <div class="flex flex-row w-full gap-12 mt-14">
+          <div class="md:flex-[7.5]">
+            <div class="mb-3 font-openSans text-gray-300">
+              <button class="uppercase hover:text-black transition-all font-2xl text-black mr-4 "
+                data-tab-target="#tab1" data-i18n="detail.special_feature">
+              </button>
+              <button class="uppercase hover:text-black transition-all font-2xl  mr-4" data-tab-target="#tab2"
+                data-i18n="detail.product-info">
+              </button>
+              <button class="uppercase hover:text-black transition-all font-2xl" data-tab-target="#tab3"
+                data-i18n="detail.review">
+              </button>
+            </div>
+            <hr>
+            <div class="mt-14">
+              <div id="tab1" class="tab-content text-gray-700">
+                <div class="flex gap-20 flex-col-reverse md:flex-row md:justify-between md:items-center mt-12">
+                  <div class="">
+                    <div class="">
+                      <h1 class="font-bold uppercase" data-i18n="detail.feature_title"></h1>
+                      <hr class="w-16">
+                    </div>
+                    <p class="mt-5 text-sm text-gray-500" data-i18n="detail.feature_des"></p>
+                  </div>
+                  <img src="../../assets/images/highlight-1.png" alt="">
```

Comment:
```suggestion
                  <img src="../../assets/images/highlight-1.png" alt="">
```
Sau này gắn data từ api, e bổ sung thêm alt nữa nha